component/src/main/java/org/cbioportal/genome_nexus/component/annotation/CanonicalTranscriptResolver.java

```diff
@@ -25,13 +25,15 @@ public TranscriptConsequence resolve(VariantAnnotation variantAnnotation)
     {
         List<TranscriptConsequence> transcripts = new ArrayList<>();
 
-        for (TranscriptConsequence transcript : variantAnnotation.getTranscriptConsequences())
-        {
-            if (transcript.getTranscriptId() != null &&
-                transcript.getCanonical() != null &&
-                transcript.getCanonical().equals("1"))
+        if (variantAnnotation.getTranscriptConsequences() != null) {
```

Comment:
@leexgh personally find it easier to read when `getTranscriptConsequences()` is assigned to a variable and then used in this null check and subsequently to iterate over instead of calling the same method twice